cmd/image-loader/main.go

```diff
@@ -172,7 +198,7 @@ func processImages(ctx context.Context, log *zap.SugaredLogger, dryRun bool, ima
 	return nil
 }
 
-func getImagesForVersion(log *zap.SugaredLogger, version *kubermaticversion.Version, addonsPath string) (images []string, err error) {
+func getImagesForVersion(ctx context.Context, log *zap.SugaredLogger, version *kubermaticversion.Version, addonsPath string) (images []string, err error) {
```

Comment:
`ctx` is not used in `getImagesForVersion`, weird this was not reported as lint violation